dissect/target/plugins/os/windows/locale.py

```diff
@@ -0,0 +1,56 @@
+from dissect.target.plugin import Plugin, export, internal
+from dissect.target.helpers.record import TargetRecordDescriptor
+
+WindowsKeyboardRecord = TargetRecordDescriptor(
+    "windows/keyboard",
+    [
+        ("string", "layout"),
+        ("string", "id"),
+    ],
+)
+
+
+class LocalePlugin(Plugin):
+    def __init__(self, target):
+        super().__init__(target)
+        self.LANG_DICT = {
+            lang.name: lang.value
+            for k in self.target.registry.key("HKLM\\SYSTEM\\CurrentControlSet\\Control\\Keyboard Layout\\DosKeybCodes")
```

Comment:
I actually don't know if this key is always present, but what I would suggest is to first check if the key exists. Else all function within the locale plugin will not be available.

Same holds for the other keys used with respect to checking for their existence ;)